pom.xml

```diff
@@ -56,5 +59,17 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>io.jenkins</groupId>
+      <artifactId>configuration-as-code</artifactId>
+      <version>${version.configuration-as-code}</version>
+      <optional>true</optional>
+    </dependency>
+    <dependency>
```

Comment:
you only need configuration-as-code plugin here, -support is useless

Reply:
Fixed. was put by mistake...